data/encounter/savage.json

```diff
@@ -0,0 +1,28 @@
+{
+    "names": [
+        "edenPromise",
+        "edenVerse",
+        "edenGate",
+        "oldTiers"
+    ],
```

Comment:
I *think* these might not be necessary, since I think you can do the following in hugo, but we should probably brainstorm on using `range` with hugo on data to make this auto-iterate. I'll take a look a bit later, but if this ends up being required, then it's required.

Reply:
Sounds good :)

Reply:
@nonowazu I made a refactor of the data structure here to accomodate the added code in the `admin/config.yml`. Please let me know if I missed anything (first time doing the config.yml stuff)